src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -0,0 +1,337 @@
+              </head>
+              <body>
+                <div class="image-text-container nr-block">
+                  <img src="https://example.com/image.png" alt="Image description">
+                </div>
+              </body>
+              </html>
+              ```
+              
+            </Collapser>
+            </CollapserGroup>
+
+    * **Ignore** means user input will be ignored entirely. For example, if you want to ignore sensitive information like a user's email address and password, you could ignore the element that contains those fields.
+
+
+
+        <CollapserGroup>
+        <Collapser
+        id="ignore-user-data"
+        title="How to ignore user data"
+    >
+        Add the CSS class `nr-ignore` to the input field’s class name. For example, if you want to ignore sensitive information on a billing page, add `class="nr-ignore"` to `<input type>`:
+
+          ```html
+          <div class="sensitive-information">
+            <h2>Sensitive Information</h2>
+            <p>Credit card number: <input type="number" class="nr-ignore" id="creditCardNumber"></p>
+            <p>Expiration date: <input type="number" class="nr-ignore" id="expirationDate"></p>
+            <p>CVV code: <input type="number" class="nr-ignore" id="cvvCode"></p>
+          </div>
+          ```
+    </Collapser>
+    </CollapserGroup>
+
+<Callout variant="tip">
+
+  Always check with your privacy teams if you decide to customize settings. Exposing user input can expose sensitive data.
+</Callout>
+
+  </Step>
+
+      <Step>
+    ### (For copy/paste deployed agents) Manually update the JavaScript snippet [#update-snippet]
+
+    If you deployed the browser agent using the copy/paste method, you'll need to copy/paste from the UI again. This is because when you enable session replay, the JavaScript snippet updates to include the new feature flag. 
+
+    To update your agent code:
+
+    1. Go to the **Application Settings** page and copy the JavaScript snippet.
+
+        <img
+            title="Copy/paste installation method"
+            alt="Screenshot of the application settings page."
+            src={browserApplicationSettingsPage}
+        />
+
+    2. Paste it in your code where the old snippet was.
+    3. Deploy your app.
+
+  </Step>
+
+
+
+<Step>
+
+### View session replays in New Relic [#view-data]
+
+In the UI, you can find session replays in two places:
+
+  * On the **Session replay** page:
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Browser**.
+        2. Select your browser app.
+        3. In the left-hand menu, scroll down and click **Session replay**.
+        4. Click into each replay to view session details.
+
+        <img
+            title="Session replay"
+            alt="screenshot of the session replay page in browser"
+            src={browserSessionReplayPage}
+        />
+
+
+
+  * On the **Errors inbox** page:
```

Comment:
This seems very useful from a user-flow perspective. I'd add a mention that you can access sessions repalys directly from errors somewhere in the intro.